brickschema/brickify/src/handlers/Handler/Handler.py

```diff
@@ -0,0 +1,152 @@
+    ):
+        """
+        Handler class responsible for performing end to end conversion
+        (including ingestion, translation, and clean up).
+
+        :param source: A filepath/URL
+        :param input_format: Input format of the file
+        :param module_path: Path to default template files in the package ([<dot-separate-module-path>, <template-filename>])
+        :param config_file: Custom conversion configuration file
+        """
+        self.graph = rdflib.Graph()
+        self.source = source
+        self.input_format = input_format
+        if config_file:
+            with open(config_file, "r") as config:
+                self.config = load_config(config, config_file)
+        elif module_path:
+            config_file = module_path[-1]
+            with importlib_resources.path(*module_path) as data_file:
+                with open(data_file, "r") as config:
+                    self.config = load_config(config, config_file)
+        else:
+            typer.echo(
+                typer.style(
+                    "[ERROR] No configuration specified!",
+                    fg=typer.colors.RED,
+                )
+            )
+
+    def update_namespaces(
+        self,
+        building_prefix=None,
+        building_namespace=None,
+        site_prefix=None,
+        site_namespace=None,
+    ):
+        """
+        Updates prefixes and namespaces from the templates for the site and the building before conversion.
+
+        :param building_prefix: Building prefix (default: bldg)
+        :param building_namespace: Building namespace (default: https://example.com/bldg#)
+        :param site_prefix: Site prefix (default: site)
+        :param site_namespace: Site namespace (default: https://example.com/site#)
+        """
+        self.config["namespace_prefixes"][building_prefix] = building_namespace
+        self.config["namespace_prefixes"][site_prefix] = site_namespace
+        bind_namespaces(self.graph, self.config["namespace_prefixes"])
+        for operation in self.config["operations"]:
+            if "query" in operation:
+                operation["query"] = re.sub(
+                    "bldg:", f"{building_prefix}:", operation["query"]
+                )
+                operation["query"] = re.sub(
+                    "site:", f"{site_prefix}:", operation["query"]
+                )
+            if "data" in operation:
+                operation["data"] = re.sub(
+                    "bldg:", f"{building_prefix}:", operation["data"]
+                )
+                operation["data"] = re.sub(
+                    "site:", f"{site_prefix}:", operation["data"]
+                )
+            if "template" in operation:
+                operation["template"] = re.sub(
+                    "bldg:", f"{building_prefix}:", operation["template"]
+                )
+                operation["template"] = re.sub(
+                    "site:", f"{site_prefix}:", operation["template"]
+                )
```

Comment:
Do these lines of code mean that there are certain fixed prefixes that brickify assumes or will always output? Or are those configurable?

Reply:
They are configurable. These lines of code replace the default namespaces and prefixes (`bldg:`, `site:`) with the ones specified (like `ghc:`, `cmu:`) as optional args. If these args are not provided, the output brick graph will have entities like "bldg:AHU_1" instead of "ghc:AHU_1".

Reply:
Oh, ok! Thanks for the clarification